problems/iterating_objects/problem.txt

```diff
@@ -0,0 +1,57 @@
+Javascript has poor standard library. It lacks many functions available to programer in other script languages like ruby or python.
+We need to include additional libraries which implements those function.
+jQuery itself provides us with few utility functions.
+In this exercise we will explore two iteration methods, '$.each()' and '.each()'.
+First one is applied to arrays, it takes two arguments, array to iterate and iteration function.
+
+{{ code }}
+  var myArray = [1, 2, 3];
+  $.each(array, function(index, element) {
```

Comment:
@pokorson actually `JS` since `ES5` has a similar function to `$.each` and its called `Array.prototype.forEach`. Maybe good idea is to mention that it is kind of similar solution